src/db/services/kuzu-transaction-manager.ts

```
@@ -0,0 +1,224 @@
+  async transaction<T>(
+    transactionBlock: (tx: {
+      executeQuery: (query: string, params?: Record<string, any>) => Promise<any>;
+    }) => Promise<T>,
+  ): Promise<T> {
+    if (!this.connectionManager.isConnected()) {
+      await this.connectionManager.initialize();
+    }
+
+    const transactionId = `tx-${Date.now()}-${Math.random().toString(36).substr(2, 9)}`;
+    const logger = this.createOperationLogger('transaction', { transactionId });
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)